modules/check_in/app/controllers/check_in/v1/patient_check_ins_controller.rb

```diff
@@ -4,40 +4,11 @@ module CheckIn
   module V1
     class PatientCheckInsController < CheckIn::ApplicationController
       def show
-        check_in = CheckIn::PatientCheckIn.build(uuid: params[:id])
```

Comment:
I might be a bit of a sticker here too, but wondering if you can remove v1 incrementally to avoid the dangerbot failure. I'm thinking the first iteration can be the changes to the controllers + corresponding specs, then probably the services classes (broken up into separate PRs)

Reply:
I agree that it can be done that way but was hoping to avoid having to create 4 or 5 separate PRs, one of which might still go above the 500 line limit. Is there a way to get an exception for this, and if not I'll go ahead and break it up.

Reply:
Yeah lets go ahead and do it... this one makes sense for it to all go out together. Did you see my question about the apidocs_controller?